src/rpc/server.h

```diff
@@ -17,6 +17,8 @@
 
 #include <univalue.h>
 
+#include <rpc/rpccategory.h>
```

Comment:
Move this after ` #include <rpc/protocol.h>`

Reply:
doing in the next commit, thanks